webpack.config.babel.js

```diff
@@ -3,7 +3,7 @@ import { join } from 'path';
 const include = join(__dirname, 'src');
 
 export default {
-  entry: './src/index.js',
```

Comment:
Aqui é só questão de gosto, mas eu costumo omitir o `js` se for `index`, mas os 2 funcionam.